### ISSUE TITLE
Update critical insights notification

### ISSUE THREAD

**Issue by fiveisprime** [microsoft/vscode-azureappservice#1191]
It's too busy and difficult to read, I'd like to see something like

```
Critical issues found for "<app-name>".
<error-preview>
Click to access diagnostics for recommendations.

<Open Diagnostics>
```

Is there any way to open to a better location?

**Comment by EricJizbaMSFT** [comment]
I think this will be fixed by https://github.com/microsoft/vscode-azureappservice/pull/1444, so I put it in the current milestone

**Comment by nturinski** [comment]
Fixed by #1444
